NorthstarDLL/scripts/client/cursorposition.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pch.h"
+#include "squirrel/squirrel.h"
+#include "util/wininfo.h"
+
+ADD_SQFUNC("vector ornull", NSGetCursorPosition, "", "", ScriptContext::UI)
+{
+	POINT p;
+	if (GetCursorPos(&p) && ScreenToClient(*g_gameHWND, &p))
```

Comment:
is there much benefit using winapi to get this rather than finding it using a game func of some kind? game probably keeps track of this too

Reply:
a) idk how to decompile the game
b) idk how to call native functions using offsets
c) I have 0 c++ experience